### PR TITLE
Create initial ci-cd starter

### PR DESCRIPTION
**Pull request by sherl0cks**
#### What does this PR do?
Provides an ease to use example that sets up a baseline java CI/CD environment in OpenShift by:
- [a set of openshift templates](https://github.com/sherl0cks/examples/tree/a4dd560cd6f4963a8b62b712bdd4335e646f828a/ci-cd-starter/files) describing the state we desire
- [an ansible vars file](https://github.com/sherl0cks/examples/blob/a4dd560cd6f4963a8b62b712bdd4335e646f828a/ci-cd-starter/vars/ci-cd-starter-vars.json) that describes how to apply the templates using [ansible-stacks](https://github.com/rht-labs/ansible-stacks)
- a [run.sh](https://github.com/sherl0cks/examples/blob/a4dd560cd6f4963a8b62b712bdd4335e646f828a/ci-cd-starter/run.sh) which pulls down [ansible-stacks](https://github.com/rht-labs/ansible-stacks) and runs a playbook using ansible-stacks roles

#### Is there a relevant Issue open for this?
No

#### Are there any other relevant resources that should be reviewed as well?
Yes. The following PR's need to be merged, and then this PR updated, before we can merge it:

- https://github.com/rht-labs/ansible-stacks/pull/104
- https://github.com/rht-labs/openshift-jenkins-s2i-config/pull/28

#### Who would you like to review this?
/cc @oybed @sabre1041 @etsauer

### CODE REVIEW

File: ci-cd-starter/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+4. The build should finish compiling the app, deploying to nexus, and creating a container image. It should now wait for confirmation to deploy the image. Approve the deployment and check that the deployment in the `Dev` project completes. If it fails, then something is wrong - open a ticket in this repo.
+5. Navigate to the Nexus webpage and click on browse -> components -> labs-snapshots. You should see an entry for "automation-api." If not, something went wrong - open a ticket in this repo.
+
+If you prefer, you can actually do the above steps by using the new Builds -> Pipelines tab in OpenShift Console as well. Be advised that "Build #1" in the Pipelines view may show build started, when in fact the build is not running because Jenkins redeployed. In this case, you'll need to kick a new build.
+
+## Conventions in Use
```
Comment (InfoSec812) [inline ci-cd-starter/README.md:68]: Why does the numbering start at 2?

Comment (InfoSec812) [reply ci-cd-starter/README.md:68]: Nevermind, it's not numbering..

**Review by InfoSec812 — APPROVED** [review]
Looks pretty good to me. I don't see any issues and I didn't have any issues using it.

File: ci-cd-starter/README.md (hunk 1 above)
Comment (sherl0cks) [reply ci-cd-starter/README.md:68]: The build numbering in OCP or Jenkins? Or the numbering in the README?

Comment (InfoSec812) [reply ci-cd-starter/README.md:68]: I thought that those were numbered items in a list, but it was not.

**Pull request merged**